src/components/UpdatePassword/UpdatePassword.tsx

```diff
@@ -0,0 +1,111 @@
+import { useState } from "react";
+
+export function UpdatePassword() {
+  const [showModal, setShowModal] = useState(true);
+  const [password, setPassword] = useState("");
+  const [rpassword, setRPassword] = useState("");
+
+  const isFormValid = () => {
+    return rpassword === password && isSecurePassword();
+  };
+
+  const isSecurePassword = () => {
+    return password.length >= 8;
+  };
+
+  return (
+    <>
+      <button
+        className="bg-white-600 mb-1 mr-1 rounded px-6 py-3 text-sm font-bold uppercase text-black shadow outline-none transition-all duration-150 ease-linear hover:shadow-lg focus:outline-none active:bg-pink-600"
+        type="button"
+        onClick={() => setShowModal(true)}
+      >
+        Update Password
+      </button>
+      {showModal ? (
+        <>
+          <div className="fixed inset-0 z-50 flex items-center justify-center overflow-y-auto overflow-x-hidden outline-none focus:outline-none">
+            <div className="relative mx-auto my-6 w-auto max-w-3xl">
+              {/*content*/}
+              <div className="relative flex w-full flex-col rounded-lg border-0 bg-white shadow-lg outline-none focus:outline-none">
+                {/*header*/}
+                <div className="flex items-start justify-between rounded-t border-b border-solid border-slate-200 p-5">
+                  <h3 className="text-3xl font-semibold text-blue-600">
+                    Modal Title
+                  </h3>
+                  <button
+                    className="float-right ml-auto border-0 bg-transparent p-1 text-3xl font-semibold leading-none text-black opacity-5 outline-none focus:outline-none"
+                    onClick={() => setShowModal(false)}
+                  >
+                    <span className="block h-6 w-6 bg-transparent text-2xl text-black opacity-5 outline-none focus:outline-none">
+                      ×
+                    </span>
+                  </button>
+                </div>
+                {/*body*/}
+                <div className="relative flex-auto p-6">
+                  {/*Contraseña nueva*/}
+                  <div className="mb-3 pt-0">
+                    <input
+                      type="password"
+                      placeholder="Contraseña nueva"
+                      className="relative w-full rounded border border-slate-300 bg-white bg-white px-3 py-3 text-sm text-slate-600 placeholder-slate-300 outline-none focus:outline-none focus:ring"
+                      value={password}
+                      onChange={(e) => setPassword(e.target.value)}
+                    />
+                    {/*Mensaje de contraseña insegura*/}
+                    {!isSecurePassword() ? (
+                      <span className="mr-1 inline-block rounded bg-red-200 px-1 py-0 text-xs font-semibold uppercase uppercase text-red-600 last:mr-0">
```

Comment:
For the alerts we are using the following library:
`https://react-hook-form.com/`